crates/cli_bin/tests/snapshots/parse__correct_variable_ranges_in_snippet_with_multiple_contexts.snap

```diff
@@ -20,18 +20,11 @@ variables:
   - name: $body
     scopedName: 11_0_$body
     ranges:
-      - start:
-          line: 1
-          column: 16
-        end:
-          line: 1
-          column: 21
-        startByte: 15
-        endByte: 20
+      - start: 15
```

Comment:
I thought you were leaving the external API the same? We do need to make sure these ranges aren't changed, because they're used to highlight metavariables in the Studio and Monaco expects lines/columns.